hardhat.config.ts

```diff
@@ -105,12 +104,8 @@ const config: HardhatUserConfig = {
   namedAccounts: {
     deployer: {
```

Comment:
I think we have to add the owner for each network, in most cases it would be the crucible multsig.

For simplicity there's no need to add it in this PR